portal_spa/src/pages/OrganizationsListPage.tsx

```diff
@@ -405,7 +420,11 @@ export default function OrganizationsListPage() {
                         {org.name_en}
                       </Link>
                     </TableCell>
-                    <TableCell>{org.trust_model}</TableCell>
+                    <TableCell>
+                      {org.trust_models && org.trust_models.length > 0
+                        ? org.trust_models.map((tm) => tm.name).join(", ")
+                        : "-"}
```

Comment:
The suggestion below will have the same behaviour but might be more readable. It makes use of nullish-coalescing (`?.`) and providing a fallback value denoted with `||`. This is approach is not for every use-case though, but for small snippets like below it's perfect.

```suggestion
                      {trust_models?.map((tm) => tm.name).join(", ") || "-"}
```